### DISCUSSION TITLE
How to add row level security rules and policies to source control

### DISCUSSION THREAD

**Discussion by burggraf** [supabase/supabase#946] [Questions] [answered]
In Firebase Firestore, we have a file called "firestore.rules" we keep under source control in our project which we deploy with:  "firebase deploy --only firestore:rules".  How do we accomplish something similar with Supabase?

**Comment by steve-chavez** [comment, marked as answer]
Hey @burggraf,

We're working on this on https://github.com/supabase/cli/pull/2.

Basically, you'll have your SQL code(policies, grants, tables, etc) on files and we'll let you deploy to production with a command(which will also generate a db migration for you automatically).

For now, you could structure your SQL in a similar way to this [project](https://github.com/Delibrium/delibrium-postgrest/blob/master/database/model/user/permissions.sql).

**Reply by burggraf** [reply]
Awesome!  Any idea when the cli will be available?

**Reply by silentworks** [reply]
It will be available next week at Supabase Launch Week. https://twitter.com/supabase_io/status/1375095676552495116

**Reply by burggraf** [reply]
Looking forward to it.  A lot of really important information is coming next week and will help us make a lot of upcoming dev decisions.  Thanks!
